automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/WorkspaceApiSpec.scala

```diff
@@ -0,0 +1,71 @@
+package org.broadinstitute.dsde.test.api.orch
+
+import java.time.Instant
+import java.util.UUID
+
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.config.{Credentials, UserPool}
+import org.broadinstitute.dsde.workbench.fixture.{BillingFixtures, WorkspaceFixtures}
+import org.broadinstitute.dsde.workbench.service.{AclEntry, Orchestration, RestException, WorkspaceAccessLevel}
+import org.scalatest.concurrent.{Eventually, ScalaFutures}
+import org.scalatest.time.{Minutes, Span}
+import org.scalatest.{FreeSpec, Matchers}
+import spray.json._
+import DefaultJsonProtocol._
+
+class WorkspaceApiSpec extends FreeSpec with Matchers with ScalaFutures with Eventually
+  with BillingFixtures with WorkspaceFixtures {
```

Comment:
is `ScalaFutures` or `Eventually` used in test?
sometimes google bucket is not ready immediately. `waitForBucketReadAccess` is used.